src/components/Property/WorkOrdersHistory/WorkOrdersHistoryTable.js

```diff
@@ -37,44 +39,117 @@ const WorkOrdersHistoryTable = ({
     }
   }
 
-  return (
-    <>
-      <h2 className="lbh-heading-h2">{tabName}</h2>
+  const handleChange = (e) => {
+    setTradeToFilterBy(e.target.value)
+  }
```

Comment:
What if, instead of passing in the event object, you pass in the trade as a string (or null)? Then call it something like onSelectTrade?